README.markdown

```diff
@@ -59,6 +59,32 @@ If you like the plugin, consider rating it on [vim.org](http://www.vim.org/scrip
   end
   ```
 
+- Python multiline strings (tries to guess SQL syntax)
+
+  ``` python
+  sql_query = """
+      SELECT name
+      FROM "Students"
+      WHERE age > 10
+  """
+
+  print("""
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+  tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+  quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+  cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+  proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+  nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+  fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+  culpa qui officia deserunt mollit anim id est laborum.
+  """)
+  ```
```

Comment:
Could we have a shorter example for this print? Maybe we could have just one lorem block with a paragraph break in the middle, around "Duis"?